src/lib/assets/deploy.ts

```diff
@@ -84,18 +85,24 @@ export async function deploy(assets: Assets, force: boolean, webhookTimeout?: nu
     throw new Error("No code provided");
   }
 
-  await setupRBAC(name, assets.capabilities, force);
+  await setupRBAC(name, assets.capabilities, force, assets.config.rbacMode, assets.config.rbac);
```

Comment:
Why do we destructure the `assets.config` object here instead of passing in `assets.config`?

Reply:
We could do either way, I was curious if there was something about splitting them up that made more sense at first